nexus/src/app/oximeter.rs

```diff
@@ -226,7 +226,7 @@ impl super::Nexus {
     /// be queried.
     /// * `criteria`: Any additional parameters to help narrow down the query
     /// selection further. These parameters are passed directly to
-    /// [oximeter::db::Client::select_timeseries_with].
+    /// [oximeter-db::client::select_timeseries_with].
```

Comment:
wat -- I did not know you were allowed to use hyphens here

Reply:
Yeah, interestingly, if I use `oximeter_db` it complains:

```
error: unresolved link to `oximeter_db::client::select_timeseries_with`
   --> nexus/src/app/oximeter.rs:229:10
    |
229 |     /// [oximeter_db::client::select_timeseries_with].
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `client` in module `oximeter_db`
```

I guess rustdoc must be trying the first name as "either a crate name or a module".